drivers/sensor/vl53l0x/vl53l0x.c

```diff
@@ -83,7 +83,8 @@ static int vl53l0x_channel_get(struct device *dev,
 			val->val1 = 0;
 		}
 	} else {
-		val->val1 = drv_data->RangingMeasurementData.RangeMilliMeter;
+		val->val1 = drv_data->RangingMeasurementData.RangeMilliMeter / 1000;
+		val->val2 = (drv_data->RangingMeasurementData.RangeMilliMeter % 1000) * 1000;
 	}
 	val->val2 = 0;
```

Comment:
Need to move this line in if (chan == SENSOR_CHAN_PROX) condition as val2 is now used. (line 85 for example)

Reply:
D'oh, you caught me, I just wanted to push something before a meeting start for CI to churn it ;-). I'll make sure to actually test on disco_l475_iot1

Reply:
Fixed